packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -18,19 +18,24 @@ void main() {
     ScrollController controller, {
     Axis axis = Axis.vertical,
     bool reverse = false,
+    TextDirection textDirection = TextDirection.ltr,
   }) {
     final AxisDirection axisDirection;
```

Comment:
It's possible that someone will add another case to the switch statement, but not assign `axisDirection` a value. To make sure that it always has a value before it's used you might want to make this a `late final` variable.

Reply:
I don't fully understand, I think you might made a mistake there, it is actually the reverse currently no one can add a switch case without assigning `axisDirection` if I add `late` the compiler won't complain if `axisDirection` isn't assigned
@a-wallen

Reply:
If it's late, but the value is null, the program will throw a runtime error. But you are right, that it's probably better that this is a compile-time error and not a runtime error. 

Thanks for the clarification.

Reply:
Thank you.